test/Sentry.Tests/Internals/ILSpy/SingleFileAppTests.cs

```diff
@@ -30,7 +30,9 @@ static SingleFileAppTests()
         InValidBundleFile = Path.Combine(testRoot, invalidBundle);
     }
 
-#if NET8_0
+#if NET9_0
+    private static string TargetFramework => "net9.0";
+#elif NET8_0
```

Comment:
@bruno-garcia [commented here](https://github.com/getsentry/sentry-dotnet/pull/3513#discussion_r1800279383):

> A revew for line 45. Instead of `throw` which will cause it to compile and fail at runtime, use:

```
#error "Target Framework not yet supported for single file apps"
```